core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -179,7 +179,7 @@ public void onNext(HttpObject o) {
                 final ResponseHeadersBuilder newHeaders = fillAdditionalHeaders(headers, additionalHeaders);
 
                 if (endOfStream && !additionalTrailers.isEmpty()) {
-                    newHeaders.setIfAbsent(additionalTrailers);
+                    newHeaders.set(additionalTrailers);
```

Comment:
What happens if a user sets `:status` using additional headers?
We should allow it? or need to check `status.isContentAlwaysEmpty()` after this?

Reply:
That's a good point. I guess we need to prohibit setting some pseudo headers.

Reply:
Now filters out some blacklisted headers. Thanks!